docs/source-2.0/guides/model-validation-examples.rst

```diff
@@ -0,0 +1,648 @@
+=========================
+Model Validation Examples
+=========================
+
+:ref:`validator-definition` can be used to lint Smithy models to help avoid
+common pitfalls and bugs as well as to enforce a common style for APIs
+designed with Smithy.
+
+This document provides a kind of "cookbook" of examples for using,
+suppressing, and modifying Smithy Validators.
```

Comment:
I'm guilty of this often, but we should use qualifiers (`kind of`, `somewhat`, etc.) sparingly in docs. I'd suggest just removing -> 'provides a "cookbook" of examples'